src/Components/Facility/FacilityHome.tsx

```diff
@@ -58,6 +58,8 @@ export const FacilityHome = ({ facilityId }: Props) => {
   const [openDeleteDialog, setOpenDeleteDialog] = useState(false);
   const [editCoverImage, setEditCoverImage] = useState(false);
   const [coverImageEdited, setCoverImageEdited] = useState(false);
+  const [coverImageLoaded, setCoverImageLoaded] = useState(false);
+  const [coverImageUrl, setCoverImageUrl] = useState<string | undefined>("");
```

Comment:
And these unnecessay useStates too could be removed that way.

Reply:
Well, coverImageLoaded is set based on the image load (addressing https://github.com/ohcnetwork/care_fe/issues/8480#issue-2505388985). Can remove the second one though.

Reply:
but that state is being used only to decide whether the tooltip should be shown or not right? but when we are going to remove the tooltip anyhow, this state is not needed in the first place right?

Reply:
> but that state is being used only to decide whether the tooltip should be shown or not right? but when we are going to remove the tooltip anyhow, this state is not needed in the first place right?

Yep, but are we removing the tool tip entirely? I thought it was to remove tool tip on successful image load.

> yes, but ensure the loading state is shown in the image edit modal till it completes.

Hmm, well the modal closes immediately, so shouldn't be an issue. On a side note, we could use the tool tip here (and remove the coverImageLoaded state).

<img width="269" alt="image" src="https://github.com/user-attachments/assets/648199fa-8b31-4b44-ae9b-6175e944d629">